packages/form-js-editor/src/features/properties-panel/entries/RepeatableEntry.js

```diff
@@ -0,0 +1,52 @@
+import { zeroPositiveIntegerEntryFactory, simpleBoolEntryFactory } from './factories';
+
+export default function RepeatableEntry(props) {
+  const {
+    field,
+    getService
+  } = props;
+
+  const {
+    type
+  } = field;
+
+  const formFieldDefinition = getService('formFields').get(type);
+
+  if (!formFieldDefinition || !formFieldDefinition.config.repeatable) {
+    return [];
+  }
+
+  const entries = [
+    zeroPositiveIntegerEntryFactory({
+      id: 'defaultRepetitions',
+      path: [ 'defaultRepetitions' ],
+      label: 'Default number of items',
+      props
+    }),
+    simpleBoolEntryFactory({
+      id: 'allowAddRemove',
+      path: [ 'allowAddRemove' ],
+      label: 'Allow add/delete items',
+      props
+    }),
+    simpleBoolEntryFactory({
+      id: 'disableCollapse',
+      path: [ 'disableCollapse' ],
+      label: 'Disable collapse',
+      props
+    })
+  ];
+
+  if (!field.disableCollapse) {
+    const nonCollapseItemsEntry = zeroPositiveIntegerEntryFactory({
```

Comment:
I should be only greater than zero, no? 0 basically means disabled

Reply:
Yup, I fixed that and also limited the default number of elements to 20 max because otherwise writing 2000 into it is an instant crash. Though the dynamic list holds up to 100 elements with 5 children with some hide conditions pretty well. Although it starts tanking in the 200s but that's enough for now. Plenty of performance improvements we want to have a look at anyways.